recipes/trolly/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "trolly" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "67c5a5644a98b56c52fcdc8c23d473a6c5f6b6f54cab29950e627a2b81aa0ca3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.python.org/packages/eb/0b/79ebf114bb81c4f942bf5af617a3c602440bfbe9dce86f3b4bbbaebef2e5/Trolly-0.2.2.zip
```

Comment:
Please use the canonical url from the example `meta.yaml`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18